addons/website_event/models/event_tag_category.py

```diff
@@ -10,3 +11,11 @@ class EventTagCategory(models.Model):
 
     def _default_is_published(self):
         return True
+
+    def default_get(self, fields_list):
```

Comment:
`default_get` should be under the fields, it's the `_default_...` methods that should be above the fields, because we need to be able to reference them directly for the fields.